### DISCUSSION TITLE
arco design pro 创建项目为啥使用的是pnpm

### DISCUSSION THREAD

**Discussion by Evn-clound** [arco-design/arco-design-vue#2321] [Q&A]
https://blog.csdn.net/Evn_484/article/details/130014305

**Comment by kaikaibenkai** [comment]
pnpm 好，用 pnpm 就是了

**Comment by cdwmhcc** [comment]
`npm install -g pnpm`，现在基本上都用 pnpm 呀，你用一下就知道了，很舒服。
